tests/integration/test_interface.py

```diff
@@ -374,50 +405,23 @@ def test_dataset_filter_by_year_with_full_dates(mocker, dataset_name: str):
         DATASETS.cps.name,
         DATASETS.tax_w2_1099.name,
         DATASETS.wic.name,
-        "TODO: tax_1040",
-    ],
-)
-def test_generate_dataset_with_state_noised(dataset_name: str, request, tmpdir):
```

Comment:
This test is being deleted because we already have tests that test state filtering and noise behavior across different seeds so this test seems redundant.